docs/book/src/topics/node-outbound-nat-gateway.md

```diff
@@ -0,0 +1,42 @@
+# Node Outbound Load Balancer
```

Comment:
Nat Gateway?

Reply:
actually should we combine this doc with the node outbound LB one and have one "Node Outbount" document presenting the two options?

Reply:
I tried merging the two. Please take a look.

Reply:
looks good